hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -161,28 +197,47 @@ def gen_matches() -> t.List[MatchesResult]:
             "signal_id": record.signal_id,
             "signal_source": record.signal_source,
             "updated_at": record.updated_at.isoformat(),
-            "reactions": "TODO",
+            "reactions": "Mocked",
         }
         for record in records
     ]
 
 
+class MatchDetailsMetaData(t.TypedDict):
```

Comment:
Nit... I think the official word is metadata, so `MatchDetailsMetadata` when using camel case.